wordpress-mobile/WordPress-iOS
# AB testing: implements an A/A experiment

Ready for review after #15770 is merged

Automattic Tracks: https://github.com/Automattic/Automattic-Tracks-iOS/pull/172

This PR implements the A/A experiment to validate that our ExPlat implementation is not biased.

### To test

1. Add a breakpoint in `ExPlatService.swift:70`
2. Add a breakpoint in `WPTabBarController+Swift.swift:89`
3. Run the app
4. The first breakpoint in `ExPlatService` should be hit
5. Tap the Reader tab
6. The second breakpoint should be hit and the property `variation` should be `control`

### Closing and reopening the app

1. Stop the app
2. Run it again
3. The first breakpoint in `ExPlatService` should be hit

### Coming back from the background

1. Put the app in the background
2. Put the app in the foreground
3. The first breakpoint in `ExPlatService` should not be hit

PR submission checklist:
